src/d3a/d3a_core/sim_results/file_export_endpoints.py

```diff
@@ -339,16 +339,21 @@ def _specific_row(self, slot, market):
         return []
 
 
-class SelfSufficiency:
+class KPI:
     def __init__(self):
+        self.performance_index = dict()
```

Comment:
Very minor, how about renaming it to `performance_indices`  As it contains multiple indices now.